fastlane/Fastfile

```diff
@@ -10,14 +10,40 @@
 #     https://docs.fastlane.tools/plugins/available-plugins
 #
 
+import("./CommonFastfile")
+
 default_platform(:ios)
 
+ENV["REPO_NAME"] = "purchases-ios"
+ENV["FILES_TO_UPDATE_VERSION_STRING"] = "
+  ../RevenueCat.podspec,
+  ../Sources/Misc/SystemInfo.swift,
+  ../.jazzy.yaml,
+  ../.version,
+  ../Examples/MagicWeather/MagicWeather.xcodeproj/project.pbxproj,
+  ../Examples/MagicWeatherSwiftUI/Magic Weather SwiftUI.xcodeproj/project.pbxproj,
+  ../scripts/docs/index.html,
+  ../Sources/Info.plist,
+  ../Sources/Misc/SystemInfo.swift,
+  ../Tests/BackendIntegrationTestApp/Info.plist,
+  ../Tests/BackendIntegrationTests/Info.plist,
+  ../Tests/UnitTests/Info.plist,
+  ../Tests/UnitTestsHostApp/Info.plist
+"
+ENV["GITHUB_RELEASE_UPLOAD_ASSETS"] = "RevenueCat.framework.zip,RevenueCat.xcframework.zip"
+
 platform :ios do
   before_all do
     setup_circle_ci
     update_fastlane
   end
 
+  desc "Bump version, update swift header, edit changelog, and create pull request"
+  lane :bump_ios do |options|
```

Comment:
From the conversation in https://github.com/RevenueCat/purchases-ios/pull/1719#discussion_r904202260.  This new lane uses the `bump` lane which is platform agnostic and adds a new step which will regenerate the `RevenueCat-Swift.h` header and commit and push if there were changes. Note that this will happen on a separate commit from the version bump changes, I added the version number to the commit name though, so it's easier to track changes on that file.

Reply:
That's correct!